components/login/header/README.md

```diff
@@ -0,0 +1,34 @@
+# LoginHeader
```

Comment:
should we keep this component on an already existing context, such as `topbar` or `header`? what do you think?

Reply:
This header is only for the "login" context. In the future, the cross team can decide whether to place it in another context or in its own repository.